### PR TITLE
Social updates from all

### PR DESCRIPTION
**Pull request by arunavemulapalli**
Updates from the team

### CODE REVIEW

**Comment by arunavemulapalli** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.social_fat,com.ibm.ws.security.social_fat.LibertyOP,com.ibm.ws.security.social_fat.delegated,com.ibm.ws.security.social_fat.multiProvider,com.ibm.ws.security.social_fat.oidcCertification

**Comment by arunavemulapalli** [comment]
#run-libby-bot

**Comment by arunavemulapalli** [comment]
The messages are already reviewed in #9930. I just made small updates based on the ID review in that PR.

**Comment by arunavemulapalli** [comment]
#run-libby-bot

**Comment by arunavemulapalli** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.social_fat,com.ibm.ws.security.social_fat.LibertyOP,com.ibm.ws.security.social_fat.delegated,com.ibm.ws.security.social_fat.multiProvider,com.ibm.ws.security.social_fat.oidcCertification

File: dev/com.ibm.ws.security.social/resources/com/ibm/ws/security/social/resources/SocialMessages.nlsprops [hunk 1]
```diff
@@ -66,23 +66,33 @@ ACCESS_TOKEN_MISSING_FROM_HEADERS.explanation=The social login configuration tha
...
-KUBERNETES_USER_API_RESPONSE_NOT_JSON.useraction=Verify that the user API value in the social login configuration is set to a valid Kubernetes user API URL. Check the response from the user API to see if more information is included.
+KUBERNETES_USER_API_RESPONSE_NOT_JSON.useraction=Verify that the user API value in the social login configuration is set to a valid Kubernetes user API URL. Check the response from the user API to see whether more information is included.
 
-# 0=JSON key string, 1=JSON value type (e.g. object, string, number), 2=JSON value type, 3=User API response string
-KUBERNETES_USER_API_RESPONSE_WRONG_JSON_TYPE=CWWKS5379E: The value for the key [{0}] in the Kubernetes user API response is expected to be of type {1}, but the value is of type {2}. The full user API response is [{3}]
+# 0=JSON key string, 1=JSON value type, for example, object, string, number, and so on, 2=JSON value type, 3=User API response string
+KUBERNETES_USER_API_RESPONSE_WRONG_JSON_TYPE=CWWKS5379E: The value for the key [{0}] in the Kubernetes user API response is expected to be of type {1}, but the value is of type {2}. The full user API response is [{3}].
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.security.social/resources/com/ibm/ws/security/social/resources/SocialMessages.nlsprops]: Change
"The value for the key [{0}] in the Kubernetes user API response is expected to be of type {1}, but the value is of type {2}."
To
"The value for the [{0}] key in the Kubernetes user API response is expected to be of type {1}, but the value is of type {2}."

File: dev/com.ibm.ws.security.social/resources/com/ibm/ws/security/social/resources/SocialMessages.nlsprops [hunk 2]
```diff
@@ -66,23 +66,33 @@ ACCESS_TOKEN_MISSING_FROM_HEADERS.explanation=The social login configuration tha
...
 KUBERNETES_USER_API_RESPONSE_WRONG_JSON_TYPE.explanation=The value for the specified key did not have the correct type. The response from the user API might be malformed, or the user API returned a response with an unexpected structure.
-KUBERNETES_USER_API_RESPONSE_WRONG_JSON_TYPE.useraction=Check the structure of the response from the user API. Check the response from the user API to see if an error occurred.
+KUBERNETES_USER_API_RESPONSE_WRONG_JSON_TYPE.useraction=Check the structure of the response from the user API. Check the response from the user API to see whether an error occurred.
+
+# 0=Error string from user API response
+KUBERNETES_USER_API_RESPONSE_ERROR=CWWKS5380E: The Kubernetes user API encountered an error while it was processing the access token. The error is [{0}]
+KUBERNETES_USER_API_RESPONSE_ERROR.explanation=The response from the Kubernetes user API contains an error. The access token might not be valid, or some other error occurred while processing the token.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.security.social/resources/com/ibm/ws/security/social/resources/SocialMessages.nlsprops]: Change
"The access token might not be valid, or some other error occurred while processing the token."
To
"The access token might not be valid, or some other error occurred while the token was being processed."

File: dev/com.ibm.ws.security.social/resources/com/ibm/ws/security/social/resources/SocialMessages.nlsprops [hunk 3]
```diff
@@ -66,23 +66,33 @@ ACCESS_TOKEN_MISSING_FROM_HEADERS.explanation=The social login configuration tha
...
+KUBERNETES_USER_API_RESPONSE_ERROR.useraction=Check the error that is displayed in the message. Ensure that the access token is valid and is not malformed.
+
+# 0=Social login configuration ID, 1=Value of the userNameAttribute, 2=userNameAttribute, 3=Alternative key that will be used instead
+KUBERNETES_USER_API_RESPONSE_DEFAULT_USER_ATTR_NOT_FOUND=CWWKS5381W: The social login configuration [{0}] specifies [{1}] as the value for the [{2}] attribute, but the user API response does not contain a [{1}] key. The [{3}] key, if present in the user API response, is used to determine the username instead.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.security.social/resources/com/ibm/ws/security/social/resources/SocialMessages.nlsprops]: Change
" The social login configuration [{0}] specifies [{1}] as the value for the [{2}] attribute, but the user API response does not contain a [{1}] key."
To
"The [{0}] social login configuration specifies [{1}] as the value for the [{2}] attribute, but the user API response does not contain a [{1}] key."

File: dev/com.ibm.ws.security.social/resources/com/ibm/ws/security/social/resources/SocialMessages.nlsprops [hunk 4]
```diff
@@ -481,8 +491,8 @@ RESPONSE_STATUS_MISSING_OR_ERROR=CWWKS5477E: The response status cannot be found
 RESPONSE_STATUS_MISSING_OR_ERROR.explanation=The request likely failed because it was unauthorized or missing information such as a required request parameter or header. If the response status was not null, the response status code and reason is included in the message.
 RESPONSE_STATUS_MISSING_OR_ERROR.useraction=Check the response status code in the message, if present. Verify that all required parameters and headers were included in the request.
 
-# 0=URL to which the request was submitted, 1=Error message
-RESPONSE_STATUS_UNSUCCESSFUL=CWWKS5478E: The request to the endpoint [{0}] failed. The error was: {1}
+# 0=URL to which the request was submitted, 1=HTTP response status (e.g. 404 Not Found), 2=Error message
+RESPONSE_STATUS_UNSUCCESSFUL=CWWKS5478E: The request to the endpoint [{0}] failed. The response status is [{1}]. The error is: {2}
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.security.social/resources/com/ibm/ws/security/social/resources/SocialMessages.nlsprops]: Change
"The request to the endpoint [{0}] failed."
To
"The request to the [{0}] endpoint failed."

**Comment by arunavemulapalli** [comment]
this one is delivered under different pr #9972

**Pull request closed without merging**